serviceregistry-server/src/main/docker/Dockerfile

```diff
@@ -15,26 +21,30 @@ EXPOSE 9099
 
 ENV APP_DIR=/var/lib/difi \
     APP_JAVA_PARAMS="" \
-    APP_PROFILE=dev \
-    DOCKERIZE_VERSION=v0.2.0
+    APP_PROFILE=dev
 
 RUN apk add --update \
         curl \
         tini \
         ca-certificates \
```

Comment:
Treng me Curl?

Reply:
Ca-certificates og openssl trengs dersom containerane frå biletet skal ja TLS-tilkoplingar. Eg meinar det kan fjernast til behovet ev. viser seg. @martinwam, kva trur du?

Reply:
Dersom antakingane mine stemmer, kan heile apk-add-steget fjernast.

Reply:
Ser jeg kan slette tini i alle fall, har ikke nok oversikt til å kunne uttale meg om ca-certs og openssl

Reply:
Er det noko i SR-oppsettet som krever at me installerar desse biblioteka for TLS-tilkopling, @martinwam ? Eg tvilar på det, og føresler derfor at me fjernar dei.